packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -162,6 +165,11 @@ class DefaultMutationMapperStore implements MutationMapperStore {
         );
     }
 
+    @computed
+    public get myCancerGenomeData(): IMyCancerGenomeData {
+        return getMyCancerGenomeData();
```

Comment:
is there a point to wrapping this call now? I guess speaking to aarons point, maybe you should just declare it as a constant i.e. `public myCancerGenomeData = getMyCancerGenomeData()`

Reply:
right, a constant may work better for this. It's just static data anyway.